### PR TITLE
Adding ncvis

### PR DESCRIPTION
**Pull request by alartum**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/ncvis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    - libgomp      # [linux and not aarch64]
+    - python
+    - pip
+    - numpy
+    - cython
+  run:
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux and not aarch64]
+    - python
+    - numpy
```
Comment (scopatz) [inline recipes/ncvis/meta.yaml]: Please see https://conda-forge.org/docs/maintainer/knowledge_base.html?#building-against-numpy for the proper way to link agains numpy

Comment (alartum) [reply recipes/ncvis/meta.yaml]: Fixed it, thank you

**Comment by scopatz** [comment]
Let us know when this is ready for review again

**Comment by alartum** [comment]
If the issue was only with pinning the correct version of numpy, I've already fixed it and ready for review :)

**Pull request merged**